src/main/support.js

```diff
@@ -133,19 +107,16 @@ function getSupportInfo(version, opts) {
     if (!object.deprecated || opts.showDeprecated) {
       return object;
     }
-    const newObject = Object.assign({}, object);
-    delete newObject.deprecated;
-    delete newObject.redirect;
+    // eslint-disable-next-line no-unused-vars
+    const { deprecated, redirect, ...newObject } = Object.assign({}, object);
```

Comment:
Let's enable https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings , shall we?

Reply:
I thought about it, but I thought this case 

```js
const {a, ...b} = c
```

if a and b both not used, it  will not report an error.

But I'm not sure now.. not near my laptop.

Reply:
BTW, `Object.assign` is not needed here.

Reply:
Oops, missed this one.

Reply:
Fixed.

Also enabled ignoreRestSiblings,

```js
const {a, ...b} = c
```

If a&b not used, report b is not used.
If a used b not used, report b is not used.
If b used a not used, no error.